Dockerfile

```diff
@@ -89,8 +90,8 @@ ENTRYPOINT ["./entrypoint.sh"]
 # It's just the VM image with a compressed Raspbian filesystem added
 FROM dockerpi-vm as dockerpi
 LABEL maintainer="Luke Childs <lukechilds123@gmail.com>"
-ARG FILESYSTEM_IMAGE_URL="http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-09-30/2019-09-26-raspbian-buster-lite.zip"
-ARG FILESYSTEM_IMAGE_CHECKSUM="a50237c2f718bd8d806b96df5b9d2174ce8b789eda1f03434ed2213bbca6c6ff"
+ARG FILESYSTEM_IMAGE_URL="https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2021-03-25/2021-03-04-raspios-buster-armhf-lite.zip"
```

Comment:
Can you revert this change?

We can add it in a separate PR.

Reply:
OK

Reply:
Should have the old version now (with the new one commented out to make it easier later)

Reply:
Thanks, I just meant remove it completely so there's no noise in the squashed commit. All good, resolved in https://github.com/lukechilds/dockerpi/pull/21/commits/767fa33c0d4c393178777f02b7068d17e0c98500.